src/material/schematics/ng-generate/mdc-migration/rules/tree-traversal.spec.ts

```diff
@@ -160,16 +160,16 @@ describe('#visitElements', () => {
     });
 
     it('should handle multiple unnested', async () => {
-      runAddAttributeTest('<a></a><b></b>', '<a add="val"></a><b add="val"></b>');
+      runAddAttributeTest('<a></a><strong></strong>', '<a add="val"></a><b add="val"></b>');
```

Comment:
It looks like the tests here need to be updated as well.

Reply:
For this one 
'<a add="val"></a><b add="val"></b>' ?

Reply:
For this one `'<a add="val"></a><b add="val"></b>'` ?